### ISSUE TITLE
Unable to install with mazer

### ISSUE THREAD

**Issue by LaurentMarchelli** [crivetimihai/ansible_virtualization#3]
Hi,

```
mazer install crivetimihai.virtualization
```
Is looping indefinitely, I guess there is an issue with this package as following command worked perfectly (It seems to be blocked on page 2).
```
mazer install crivetimihai.development
mazer install crivetimihai.server
mazer install crivetimihai.workstation
```
Let me know if you want more details.
Regards,

**Comment by crivetimihai** [comment]
As mazer is now deprecated, does ansible-galaxy collection install crivetimihai.virtualization solve the issue?

**Comment by crivetimihai** [comment]
Closed as duplicate of #2 

The issue seems to be with `mazer`, as `ansible-galaxy collection install crivetimihai.virtualization` works fine.

**Comment by LaurentMarchelli** [comment]
Hi Mihai,

Install did not work with 2.8.5, but it works fine with ansible 2.9.rc1.

```
$ ansible-galaxy collection install crivetimihai.virtualization -p ~/.ansible
Process install dependency map
Starting collection install process
Installing 'crivetimihai.virtualization:1.0.31' to '~/.ansible/ansible_collections/crivetimihai/virtualization'
```

Thank you very much for your help.
Best Regards,
Laurent Marchelli

**Comment by LaurentMarchelli** [comment]
However `-p ~/.ansible/collections` seems better to avoid configuration issues when using roles in the playbook. :-)
```
ansible-galaxy collection install crivetimihai.virtualization -p ~/.ansible/collections
```
